newrelic-agent/src/main/java/com/newrelic/agent/config/ConfigFileHelper.java

```diff
@@ -8,6 +8,7 @@
 package com.newrelic.agent.config;
 
 import com.newrelic.agent.DebugFlag;
+import org.jetbrains.annotations.Nullable;
```

Comment:
We probably don't want to use this annotation.

Reply:
Nah, it snuck in there when I had IJ do a refactor. Deleted.